test/OpenTelemetry.Instrumentation.Grpc.Tests/GrpcTests.server.cs

```diff
@@ -108,6 +113,86 @@ public void GrpcAspNetCoreInstrumentationAddsCorrectAttributes(bool? enableGrpcA
             Assert.StartsWith("grpc-dotnet", activity.GetTagValue(SemanticConventions.AttributeHttpUserAgent) as string);
         }
 
+        // TO DO
+        // Test to check if GrpcAspNetCore instrumentation correctly adds the attributes when it creates a sibling activity
+        [Theory(Skip = "To be run after fixing the gRPC instrumentation issue #1740")]
+        [InlineData(null)]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void GrpcAspNetCoreInstrumentationAddsCorrectAttributesWhenItCreatesNewActivity(bool? enableGrpcAspNetCoreSupport)
+        {
+            // This CompositeTextMapPropagator along with the headers passed to the client.SayHello ensure that the instrumentation creates a sibling activity
+            Sdk.SetDefaultTextMapPropagator(new CompositeTextMapPropagator(new TextMapPropagator[]
+            {
+                new B3Propagator(),
+                new TraceContextPropagator(),
+                new BaggagePropagator(),
+            }));
+            var processor = new Mock<BaseProcessor<Activity>>();
+            var tracerProviderBuilder = Sdk.CreateTracerProviderBuilder();
+
+            if (enableGrpcAspNetCoreSupport.HasValue)
+            {
+                tracerProviderBuilder.AddAspNetCoreInstrumentation(options =>
+                {
+                    options.EnableGrpcAspNetCoreSupport = enableGrpcAspNetCoreSupport.Value;
```

Comment:
This test is only applicable if GrpcAspNetCoreSupport is enabled right? So why test for the case where it is disabled?

Reply:
I followed one of the existing unit tests in the same file: `GrpcAspNetCoreInstrumentationAddsCorrectAttributes`. It seems like when `GrpcAspNetCoreSupport`  is enabled we expect the rpc.* tags as mentioned in the spec but when `GrpcAspNetCoreSupport` option is disabled we expect these two tags added to the activity: `grpc.method` and `grpc.status_code`

Reply:
Referring to this [comment](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1423/files#r514506008), we do this check to ensure that the instrumentation does not change these tags added by the framework.